### PR TITLE
[IN-2698] fix lack of remote access issue

### PR DESCRIPTION
**Pull request by nagypeterjob**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [x] I updated the corresponding tests if there were any.
- [x] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/remote-access/files/bootstrap_remote_access.sh.tmpl [hunk 1]
```diff
@@ -36,3 +36,17 @@ echo "$NGROK_CONF" | sudo tee /etc/ngrok/ngrok.yml
...
+
+# Enabling RemoteManagement access
+KICKSTART_PATH="/System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart"
+sudo "${KICKSTART_PATH}" -configure -allowAccessFor -specifiedUsers -quiet
+sudo "${KICKSTART_PATH}" -configure -allowAccessFor -allUsers -privs -all -quiet
+sudo "${KICKSTART_PATH}" -activat -quiet
+
+OS_MAJOR_VERSION=`sw_vers | sed -n 2p | awk '{ print $2 }' | cut -c1-2`
+
+if test "z$OS_MAJOR_VERSION" = z11; then
```
Comment (peterheja-bitrise) [inline roles/remote-access/files/bootstrap_remote_access.sh.tmpl:48]:
```suggestion
if [ "$OS_MAJOR_VERSION" = "11" ]; then
```

Please, use the proper built-in version.

File: roles/remote-access/files/bootstrap_remote_access.sh.tmpl [hunk 2]
```diff
@@ -36,3 +36,17 @@ echo "$NGROK_CONF" | sudo tee /etc/ngrok/ngrok.yml
...
+  # For some reason VNC on Big Sur renders a "black screen" without
+  # resetting ScreenCapture first
+  tccutil reset ScreenCapture
+fi
```
Comment (peterheja-bitrise) [inline roles/remote-access/files/bootstrap_remote_access.sh.tmpl]:
```suggestion
fi

```

File: roles/remote-access/files/bootstrap_remote_access.sh.tmpl [hunk 3]
```diff
@@ -36,3 +36,17 @@ echo "$NGROK_CONF" | sudo tee /etc/ngrok/ngrok.yml
 # install and start ngrok as a service
 sudo ngrok service install -config /etc/ngrok/ngrok.yml
 sudo ngrok service start
+
+# Enabling RemoteManagement access
+KICKSTART_PATH="/System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart"
+sudo "${KICKSTART_PATH}" -configure -allowAccessFor -specifiedUsers -quiet
+sudo "${KICKSTART_PATH}" -configure -allowAccessFor -allUsers -privs -all -quiet
+sudo "${KICKSTART_PATH}" -activat -quiet
```
Comment (bitrise-tom) [inline roles/remote-access/files/bootstrap_remote_access.sh.tmpl:44]: activate?

File: roles/remote-access/files/bootstrap_remote_access.sh.tmpl (hunk 1 above)
Comment (bitrise-tom) [reply roles/remote-access/files/bootstrap_remote_access.sh.tmpl]: i like checking strings like this, it ensures they aren't empty

File: roles/remote-access/files/bootstrap_remote_access.sh.tmpl (hunk 3 above)
Comment (nagypeterjob) [reply roles/remote-access/files/bootstrap_remote_access.sh.tmpl]: nice catch

File: roles/remote-access/files/bootstrap_remote_access.sh.tmpl (hunk 1 above)
Comment (peterheja-bitrise) [reply roles/remote-access/files/bootstrap_remote_access.sh.tmpl]: So does the other, as it gives the `[` binary a zero-length string as a argument, however, this decreases readability.

**Review by bitrise-tom — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request merged**